docs/kyma-release-notes/2023-08-31-release-notes-2.18.md

```diff
@@ -0,0 +1,26 @@
+With some of its offspring having left the parent ship as fully-grown, [independent modules](https://kyma-project.io/#/?id=kyma-modules) and more getting ready to follow suit, [Kyma’s sailing, Kyma’s sailing…](https://www.youtube.com/watch?v=FOt3oQ_k008) and still evolving. To understand its journey towards enhanced functionality, look at the latest updates and fixes.
+
+## Application Connectivity
+With this release, all 5XX codes passing through Central Application Gateway are now rewritten to `502`. The original error code is returned in a Target-System-Status header.
+
+## Telemetry
+Kyma 2.18 brings the following improvements:
+
+- We’ve [fixed the bug that caused problems scraping the metrics of the Fluent Bit component](https://github.com/kyma-project/kyma/issues/17976) to third-party vendors.
+- We‘ve added [mTLS support for TracePipeline OTLP outputs](https://github.com/kyma-project/kyma/issues/17995).
+- We‘ve [updated the following software stack](https://github.com/kyma-project/kyma/pull/18021):
+    - OTel Collector 0.83.0  
+    - Fluent Bit 2.1.8
+
```

Comment:
```suggestion

## Security 
The Cluster Users component [has been deprecated since Kyma 2.7](https://github.com/kyma-project/website/blob/main/content/blog-posts/2022-09-22-release-notes-2.7/index.md#cluster-users-component-deprecated) and will be removed with Kyma 2.19. 
This component includes predefined Kubernetes ClusterRoles such as `kyma-admin`, `kyma-namespace-admin`, `kyma-edit`, `kyma-developer`, `kyma-view`, and `kyma-essentials`. These Roles specify permissions for accessing Kyma resources that you can assign to users. For example, if you bind a user to the `kyma-admin` ClusterRole, it grants them full admin access to the entire cluster, and if you bind a user to the `kyma-view` ClusterRole, they are only allowed to view and list the resources within the cluster. 
Once the component is removed, these Roles will no longer be available for newly created clusters. This means that you won’t be able to use these predefined sets of rights and will be required to specify yourself which users or groups should have access to which of your resources. However, for clusters created before the release of Kyma 2.19, the already-defined Roles will not be deleted.

```